web/src/components/merchant/listing-details/CommittedCustomerStatuses.tsx

```diff
@@ -50,28 +54,43 @@ const CommitStatusCount = styled.div`
   font-weight: bold;
 `;
 
+const CommitStatusDetails = styled.div`
+  margin-left: 15px;
+  margin-top: 10px;
+  max-height: 60vh;
+`;
+
 interface CommitStatusStatsProps {
   colour: 'bright-red' | 'yellow' | 'green';
   label: ReactNode;
   count: number;
+  commits?: Commit[];
```

Comment:
rename to paidCommits?

Reply:
@ifhuang `CommitStatusStats` is also used for the other commit statuses, so I thought it might be better to omit the paid.